helm/prometheus-agent/Chart.yaml

```diff
@@ -4,7 +4,7 @@ name: prometheus-agent
 description: A Helm chart for Prometheus agent.
 dependencies:
   - name: prometheus-agent
-    version: 0.6.2
+    version: 0.6.3
```

Comment:
Don't forget to upgrade that when you release.

BTW could you make this a value that we could override via helm?

Reply:
How can I override a value in chart.yaml ?

Reply:
Sorry, I edited the comment. I meant the value of tolerance below